src/current/v23.2/admission-control.md

```diff
@@ -103,6 +103,18 @@ To reset the priority to the default session setting (in between background and
 SET default_transaction_quality_of_service=regular;
 ~~~
 
+### Set quality of service level for a transaction
+
+To set the quality of service level for a single [transaction]({% link {{ page.version.version }}/transactions.md %}), set the [`default_transaction_quality_of_service` session variable]({% link {{ page.version.version }}/set-vars.md %}#default-transaction-quality-of-service) for just that transaction using the [`SET LOCAL`]({% link {{ page.version.version }}/set-vars.md %}#set-a-variable-for-the-duration-of-a-single-transaction) statement inside a [`BEGIN`]({% link {{ page.version.version }}/begin-transaction.md %}) ... [`COMMIT`]({% link {{ page.version.version }}/commit-transaction.md %}) block as shown below. The valid values are `critical`, `background`, and `regular`.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BEGIN;
+SET LOCAL default_transaction_quality_of_service = 'level';
```

Comment:
I'd suggest this (per our style) or just one of the possible values you mentioned, for clarity

```suggestion
SET LOCAL default_transaction_quality_of_service = '{level}';
```

Reply:
hm, i think the `{}` braces inside of single quotes might actually be confusing, since that's the syntax for array literals.

example:
```
root@localhost:26257/defaultdb> select array[1, 2, 3]::int[];
   array
-----------
  {1,2,3}

root@localhost:26257/defaultdb> select '{1, 2, 3}'::int[];
   int8
-----------
  {1,2,3}
```

Reply:
Updated to use one of the allowed values, but added a little explanatory comment 

```
SET LOCAL default_transaction_quality_of_service = 'regular'; -- Edit to desired level
```